features/tabelwaarden.feature

```diff
@@ -0,0 +1,90 @@
+Functionaliteit: Als developer wil ik waardetabellen gebruiken in plaats van enumeraties
+  zodat ik mijn code niet hoef aan te passen als een enumeratielijst wordt aangepast die ik in mijn functionaliteit niet gebruik.
+
+  Wanneer in de registratie specifieke waarden in een enumeratie voorkomen, worden deze waarden in de API doorgegeven middels een waardetabel.
+  Wanneer in de registratie een omschrijving gekoppeld is aan een specifieke enumeratiewaarde, wordt deze omschrijving raadpleegbaar gemaakt middels de waardetabellen API.
+
+
+  Voor de enumeratie AanduidingBijzonderNederlanderschap wordt de tabel met tabelidentificatie "aanduidingbijzondernederlanderschap" opgenomen.
+  In de omschrijving wordt de volgende tekst opgenomen : "Geeft aan dat de persoon behandeld wordt als Nederlander, of dat door de rechter is vastgesteld dat de persoon niet de Nederlandse nationaliteit bezit."
```

Comment:
hier neem je de mogelijke waarden op in de fabelomschrijving. Is denk ik niet handig. Ook geeft deze tabel dit niet aan. de tabel bevat alleen mogelijke waarden.

Waarom niet "Mogelijke waarden voor bijzonder Nederlanderschap"

Ik vraag me trouwens af of in dit geval een tabel de beste oplossing is. Gaat het hier om één gegeven met twee gelijksoortige waarden? Met andere woorden zou dit niet beter opgelost moeten worden met twee booleans (behandeldAlsNederlander en vastgesteldNietNederlander)?

Reply:
Twee booleans dekken de lading niet omdat de ene waarde de andere uitsluit. ALhoewel dat vooral een vastleggings-issue is. Bij bevragen zou dit dus wel een optie zijn, alleen is voor de consumer-developer dan de informatie dat de ene waarde de andere uitsluit verloren gegaan. Ik kan niet inschatten in hoeverre dat relevant is.

Reply:
Besproken en mijn stelling klopt niet. Inmiddels met 2 booleans opgenomen.